src/helpers/user_normalizer.coffee

```diff
@@ -0,0 +1,7 @@
+class UserNormalizer
+  @normalize: (username, userInput)->
+    if not userInput? or (userInput?.toLocaleLowerCase() is 'me')
+      return '@' + username
+    '@' + userInput.replace /@*/g, ''
```

Comment:
Is there a reason why usernames are prefixed with `@`? I would prefer them to be kept as is, without any prefix or normalization, exactly as they were before.